### ISSUE TITLE
require-param, require-returns do not work for interface properties

### ISSUE THREAD

**Issue by ilyub** [gajus/eslint-plugin-jsdoc#778]
## Expected behavior

Should check all 3 functions: f, g, and h.

## Actual behavior

Checks only g and h.

Adding TSTypeAnnotation and/or TSPropertySignature contexts  does not help.

## ESLint Config

require-param, require-returns: "warn"

## ESLint sample

```ts
export interface I {
  /**
   * Description.
   */
  readonly f: (value: object) => object;

  /**
   * Description.
   */
   g(value: object): object;
}

export class C {
  /**
   * Description.
   */
  public h(value: object): object {
    return value;
  }
}
```

## Environment

- `eslint-plugin-jsdoc` version: 36.0.7

**Comment by brettz9** [comment]
Did you try `TSPropertySignature > TSTypeAnnotation > TSFunctionType`? If you search the issues, you should be able to find similar questions. If not, see also the Advanced "AST and Selectors" section of the README.

**Comment by ilyub** [comment]
That worked. Thank you.
